content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -76,7 +76,7 @@ The following example stores two strings in a Secret using the `data` field.
 To verify that the Secret was created and to decode the Secret data, refer to
 [Managing Secrets using kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#verify-the-secret).
 
-### Specify unencoded data when creating a Secret
+### Specify unencoded data when creating a Secret {#specify-unencoded-data-when-creating-a-secret}
```

Comment:
Why this change?

Reply:
Consider whether adding an anchor like h2 is helpful to link with other topics. If not, will remove it. 🤔